src/opds_dumper.cpp

```diff
@@ -41,6 +41,7 @@ OPDSDumper::~OPDSDumper()
 
 namespace
 {
+const std::string kiwix_logo_url="static/skin/kiwix_logo.png";  
```

Comment:
This is a relative URL. I doubt that it will work in all (even the most typical) cases.